cacti/datadog_checks/cacti/cacti.py

```diff
@@ -96,16 +102,17 @@ def _get_config(cls, instance):
         user = instance.get('mysql_user')
         password = instance.get('mysql_password', '') or ''
         db = instance.get('mysql_db', 'cacti')
+        port = instance.get('mysql_port')
```

Comment:
I guess `pymysql.connect` defaults to 3306 if `self.config.port` is None, but we can make that more explicit for readers here.

```suggestion
        port = instance.get('mysql_port', 3306)
```

Reply:
I prefer leaving that to the lib, same as with host. Even if I dont think mysql default port is going to change anytime soon